### ISSUE TITLE
Training beta / gamma parameters

### ISSUE THREAD

**Issue by bermanmaxim** [DrSleep/tensorflow-deeplab-resnet#66]
Hello,

I have a small question, pardon me to open an issue for this: why are the beta/gamma parameters of the batch normalizations not finetuned during training? Is this to replicate the caffe learning?

Thank you

**Comment by DrSleep** [comment]
Hi!

Yes, it is just to reproduce the caffe training settings.

**Comment by bermanmaxim** [comment]
Okay! thanks.
